config/jobs/kubernetes/sig-release/release-branch-jobs/1.27.yaml

```diff
@@ -254,13 +254,13 @@ periodics:
   - 'perfDashPrefix: kubemark-500Nodes-1.27'
   - 'perfDashJobType: performance'
 - annotations:
-    fork-per-release-cron: 0 8-20/24 * * *
+    fork-per-release-cron: ""
```

Comment:
Looks like this is getting dropped, is that ok?

Reply:
https://github.com/kubernetes/test-infra/pull/33201#discussion_r1701796786